### PR TITLE
Add runSymfonyConsoleCommand

### PR DESCRIPTION
**Pull request by wirwolf**

### CODE REVIEW

**Comment by Naktibalda** [comment]
I am not sure if this method belongs to Symfony module which simulates accessing Symfony web applications over HTTP.

**Comment by wirwolf** [comment]
Not over HTTP. I test console command by this function

File: src/Codeception/Module/Symfony.php [hunk 1]
```diff
@@ -532,6 +532,32 @@ public function grabService($service)
         }
         return $container->get($service);
     }
+    
+     /**
+     * @param string  $command
```
Comment (DavertMik) [inline src/Codeception/Module/Symfony.php]: Please add a valid description with a code example here in docblock

Comment (wirwolf) [reply src/Codeception/Module/Symfony.php]: fixed

**Comment by DavertMik** [comment]
Let's make it!

**Comment by Ravetracer** [comment]
I just tried this feature and in the first approach it failed.
After some investigation I realized, that some use statements are missing from the Symfony.php module:

use Symfony\Bundle\FrameworkBundle\Console\Application;
use Symfony\Component\Console\Input\ArrayInput;
use Symfony\Component\Console\Output\BufferedOutput;

I just upgraded the version of Codeception in composer and updated then. 
BTW: I'm using Symfony 4.2.

**Comment by wirwolf** [comment]
Yes, i found this bug today in my project.

**Pull request merged**